docs/asyncapi.yaml

```diff
@@ -115,6 +119,212 @@ components:
+                type: string
+                description: System identifier request originates from
+              value:
+                type: number
+                description: Integer net value of request, ie sum of all invoice lines
+          completedPaymentRequests:
+            type: array
+            description: Payment request after both delta calculations and automated ledger assignment
+            items:
+              type: object
+              properties:
+                agreementNumber:
+                  type: string
+                  description: Unique reference number for agreement/application
+                contractNumber:
+                  type: string
+                  description: Contract reference number of agreement
+                currency:
+                  type: string
+                  description: Currency of values in request, eg GBP
+                debtType:
+                  type: string
+                  description: Reason for recovery, optional
+                deliveryBody:
+                  type: string
+                  description: Delivery body responsible for payment
+                dueDate:
+                  type: string
+                  description: Date request should be issued from DAX in format DD/MM/YYYY
+                frn:
+                  type: number
+                  description: Firm Reference Number
+                invoiceCorrectionReference:
+                  type: string
+                  description: Original invoice number if correction to AR
+                invoiceLines:
+                  type: array
+                  description: List of Invoice lines that make up request
+                  items:
+                    type: object
+                    properties:
+                      description:
+                        type: string
+                        description: Description of what line relates to, eg gross or penalty
+                      fundCode:
+                        type: string
+                        description: Unique fund code to pay from
+                      schemeCode:
+                        type: string
+                        description: Unique scheme code for funding option
+                      value:
+                        type: string
+                        description: Integer value of invoice line
+                invoiceNumber:
+                  type: string
+                  description: Unique request for payment
+                ledger:
+                  type: string
+                  description: Target ledger for settlement, AP for Accounts Payable and AR for Accounts Receivable
+                marketingYear:
+                  type: number
+                  description: Scheme year for request
+                originalSettlementDate:
+                  type: string
+                  description: Date original invoice settled, optional
+                paymentRequestId:
```

Comment:
Id should not be returned as it means nothing and only causes confusion between micro-services when `paymentRequestId: 1` in `ffc-pay-processing` is the corresp. `paymentRequestId: 2` record in `ffc-pay-request-editor`